Gemfile

```diff
@@ -17,10 +17,16 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a
   # debugger console
   gem 'byebug'
+  gem 'rspec-rails', '~> 3.4'
   gem 'spring'
-  gem 'rspec-rails'
 end
 
 group :development do
   gem 'rubocop', '~> 0.38.0', require: false
 end
+
+group :test do
+  gem 'factory_girl_rails', '~> 4.0'
```

Comment:
The '~>' means that bundle can install any minor version greater than or equal to 4.0, so 4.6 is a legal choice. I'm not sure what's up with rubymine... is there a full callstack you can share? I do expect this is a problem with your dev environment though. (Travis CI and) I just run `bundle exec rake spec` to run the tests. I can successfully start a server with `bundle exec rails s`, too.